## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 393600`

---
### `x86_64-darwin`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>tailscale</li>
    <li>tailscale.derper</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>tailscale-gitops-pusher</li>
    <li>tailscale-nginx-auth</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>tailscale</li>
    <li>tailscale.derper</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>tailscale-gitops-pusher</li>
    <li>tailscale-nginx-auth</li>
  </ul>
</details>

---

A few tests fail on on Darwin:

```
FAIL    tailscale.com/cmd/containerboot [setup failed]
--- FAIL: TestOnTailnetDefaultAutoUpdate
FAIL    tailscale.com/ipn/ipnlocal
```

To reproduce this:

```bash
nix run 'nixpkgs#nixpkgs-review' -- pr --print-result --systems "x86_64-darwin aarch64-darwin" 393600
```

--- a/pkgs/by-name/ta/tailscale-gitops-pusher/package.nix
+++ b/pkgs/by-name/ta/tailscale-gitops-pusher/package.nix
@@ -1,10 +1,10 @@
 {
   lib,
   tailscale,
-  buildGo123Module,
+  buildGo124Module,
 }:
 
-buildGo123Module {
+buildGo124Module {
   inherit (tailscale)
     version
     src

--- a/pkgs/by-name/ta/tailscale/package.nix
+++ b/pkgs/by-name/ta/tailscale/package.nix
@@ -24,7 +24,7 @@
 }:
 
 let
-  version = "1.80.3";
+  version = "1.82.0";
 in
 buildGoModule {
   pname = "tailscale";
@@ -39,25 +39,10 @@ buildGoModule {
     owner = "tailscale";
     repo = "tailscale";
     rev = "v${version}";
-    hash = "sha256-UOz2EAUlYZx2XBzw8hADO0ti9bgwz19MTg60rSefSB8=";
+    hash = "sha256-/8MJE1Io0MRSUXiFirYDn6FH2qkWQHE7Fl3nJLjieDw=";
   };
 
-  patches = [
-    # Fix "tailscale ssh" when built with ts_include_cli tag
-    # https://github.com/tailscale/tailscale/pull/12109
-    (fetchpatch {
-      url = "https://github.com/tailscale/tailscale/commit/325ca13c4549c1af58273330744d160602218af9.patch";
-      hash = "sha256-SMwqZiGNVflhPShlHP+7Gmn0v4b6Gr4VZGIF/oJAY8M=";
-    })
-    # Fix build with Go 1.24
-    (fetchpatch {
-      url = "https://github.com/tailscale/tailscale/commit/836c01258de01a38fdd267957eeedab7faf0f4f2.patch";
-      includes = ["cmd/testwrapper/*" "cmd/tsconnect/*"];
-      hash = "sha256-e+IQB2nlJmJCzCTbASiqX2sXKmwVNXb+d87DdwTdJ+I=";
-    })
-  ];
-
-  vendorHash = "sha256-81UOjoC5GJqhNs4vWcQ2/B9FMaDWtl0rbuFXmxbu5dI=";
+  vendorHash = "sha256-SiUkN6BQK1IQmLfkfPetzvYqRu9ENK6+6txtGxegF5Y=";
 
   nativeBuildInputs = [
     makeWrapper
@@ -160,6 +145,9 @@ buildGoModule {
 
           # portlist_test.go:81: didn't find ephemeral port in p2 53643
           "TestPoller" # portlist
+
+          # Fails only on Darwin, succeeds on other tested platforms.
+          "TestOnTailnetDefaultAutoUpdate"
         ];
     in
     [ "-skip=^${builtins.concatStringsSep "$|^" skippedTests}$" ];

